sql/src/main/java/io/crate/planner/PlanNodeBuilder.java

```diff
@@ -21,77 +21,24 @@
 import io.crate.metadata.Routing;
-import io.crate.metadata.table.TableInfo;
-import io.crate.planner.node.dql.CollectPhase;
-import io.crate.planner.projection.Projection;
-import io.crate.planner.symbol.InputColumn;
-import io.crate.planner.symbol.Symbol;
 
-import javax.annotation.Nullable;
 import java.util.List;
 import java.util.Map;
 import java.util.TreeMap;
-import java.util.UUID;
 
 public class PlanNodeBuilder {
 
-    public static CollectPhase collect(UUID jobId,
-                                       TableInfo tableInfo,
-                                       Planner.Context plannerContext,
-                                       WhereClause whereClause,
-                                       List<Symbol> toCollect,
-                                       List<Projection> projections,
-                                       @Nullable String partitionIdent,
-                                       @Nullable String routingPreference,
-                                       @Nullable OrderBy orderBy,
-                                       @Nullable Integer limit) {
-        Routing routing = tableInfo.getRouting(whereClause, routingPreference);
-        return collect(jobId, tableInfo, plannerContext, whereClause, routing, toCollect, projections, partitionIdent, orderBy, limit);
-    }
-
-    public static CollectPhase collect(UUID jobId,
-                                       TableInfo tableInfo,
-                                       Planner.Context plannerContext,
-                                       WhereClause whereClause,
-                                       Routing routing,
-                                       List<Symbol> toCollect,
-                                       List<Projection> projections,
-                                       @Nullable String partitionIdent,
-                                       @Nullable OrderBy orderBy,
-                                       @Nullable Integer limit) {
-        assert !Iterables.any(toCollect, Predicates.instanceOf(InputColumn.class)) : "cannot collect inputcolumns";
-        if (partitionIdent != null && routing.hasLocations()) {
-            routing = filterRouting(routing, PartitionName.fromPartitionIdent(
-                    tableInfo.ident().schema(), tableInfo.ident().name(), partitionIdent).stringValue());
-        }
-        plannerContext.allocateJobSearchContextIds(routing);
-        CollectPhase node = new CollectPhase(
-                jobId,
-                plannerContext.nextExecutionPhaseId(),
-                "collect",
-                routing,
-                tableInfo.rowGranularity(),
-                toCollect,
-                projections,
-                whereClause);
-        node.orderBy(orderBy);
-        node.limit(limit);
-        return node;
-    }
-
-    private static Routing filterRouting(Routing routing, String includeTableName) {
+    public static Routing filterRouting(Routing routing, String includeTableName) {
```

Comment:
as this class is now doing nothing as this, maybe rename the class or delete it and move this static method somewhere else (maybe Routing.class?)